DOCS-12589-updated request security assessment (#575)
mulesoft/docs-runtime-manager: modules/ROOT/pages/penetration-testing-policies.adoc

To request a Security Assessment:

> . Navigate to: https://security.secure.force.com/securityassessments and click on *Sign-in with SecureAssess* if you don't have a Salesforce account.

Reviewer: ```suggestion
. Navigate to https://security.secure.force.com/securityassessments and click *Sign-in with SecureAssess* if you don't have a Salesforce account.
```

> . Select *New User* and register your details. Once registered and approved, you'll be able to log-in to the portal and submit your pen test request.

Reviewer: Per style ref:
```suggestion
. Select *New User* and register your details. After you are registered and approved, you can log in to the portal and submit your pen test request.
```

Reviewer: However...at this point I read further into the topic, and I sense that the information is not chronological. I wonder if you would prefer to organize this way (I shortened for convenience; feel free to add to it):

1. Navigate to...
2. If you do not already have a Salesforce account, click...
3. Select *New User*.
4. Complete the request form [[optional] with the details you want to register.] See <link>.
5. Accept the agreement and wait for automated approval.
6. If you do not receive the approval notice, check your Spam folder before contacting Salesforce.

After you are approved, you can log in to the portal and submit your pen test request.

NOTE: If you prefer, you can request approval manually by submitting the Security Assessment Request to <email>.

. Complete the request form and accept the agreement.
+
The automated system contacts you when your Security Assessment request is approved.
+
Check your Spam folder for the automated reply before contacting Salesforce.
+

> For information required to complete the request form, see the knowledge article https://help.salesforce.com/articleView?id=000322336&type=1&mode=1[Complete the Security Assessment submission form].

Reviewer: ```suggestion
For further information about completing the request form, see https://help.salesforce.com/articleView?id=000322336&type=1&mode=1[Complete the Security Assessment].
```

[NOTE]

> If preferred, you can also send your completed https://content.api.getguru.com/files/view/fbf972df-460b-4881-b84b-94d2cd3efc72[Security Assessment Request manual form] to mailto:securityassessment@salesforce.com[securityassessment@salesforce.com].

Reviewer: ```suggestion
Alternatively, send a manually completed https://content.api.getguru.com/files/view/fbf972df-460b-4881-b84b-94d2cd3efc72[Security Assessment Request] to mailto:securityassessment@salesforce.com[securityassessment@salesforce.com].
```
